### PR TITLE
Update 0422.lang

### PR DESCRIPTION
**Pull request by Oleg-Chashko**
Update #2

### CODE REVIEW

File: Langs/0422.lang [hunk 1]
```diff
@@ -28,9 +29,9 @@ Storage Available=Доступный об'єм
 TPM Version=Версія TPM
 Checking...=Йде перевірка...
 64 Bit CPU=64-бітний процессор
-64 Bit OS=64-бітна ОС
+64 Bit OS=64-бітна Операційна система
```
Comment (micwoj92) [inline Langs/0422.lang:32]: Did you check that it fits? Feel free to correct later if it seems too long.

Comment (Oleg-Chashko) [reply Langs/0422.lang:32]: > Did you check that it fits? Feel free to correct later if it seems too long.

Thanks for reminding me, I'll be sure to check it out.

Comment (Oleg-Chashko) [reply Langs/0422.lang:32]: > Did you check that it fits? Feel free to correct later if it seems too long.

@micwoj92 

Screen Shot 2021-07-11 at 21 01 00

Comment (micwoj92) [reply Langs/0422.lang:32]: Thx for checking.

**Pull request merged**